app/components/badge-maintenance.js

```diff
@@ -0,0 +1,46 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { alias } from '@ember/object/computed';
+
+export default Component.extend({
+    tagName: 'span',
+    classNames: ['badge'],
+    escapedStatus: computed('badge', function() {
+        return this.get('badge.attributes.status').replace('-', '--');
+    }),
+    none: computed('badge', function() {
+        if (this.get('badge.attributes.status') === 'none'
+            || this.get('badge.attributes.status') == undefined) {
+            return true;
+        }
+        return false;
+    }),
+    status: alias('badge.attributes.status'),
+    color: computed('badge', function() {
+        let color = 'undefined';
```

Comment:
I think this function could be simplified by removing this `color` variable and just returning from each case, something like:

```
        switch (this.get('badge.attributes.status')) {
        case 'actively-developed':
            return 'brightgreen';
```

WDYT?

I also don't think we need to explicitly handle anything not listed here; whether we set the color to the string 'undefined' or whether we don't set a color, the badge is going to be broken anyway. Thoughts?

Reply:
Yep I agree, I've gone ahead and removed handling it.